auth.go

```diff
@@ -395,15 +341,7 @@ func authenticate(
 		params.Add("roleName", sc.cfg.Role)
 	}
 
-	jsonBody, err := json.Marshal(authRequest)
-	if err != nil {
-		return
-	}
-
-	logger.WithContext(sc.ctx).Infof("PARAMS for Auth: %v, %v, %v, %v, %v, %v",
-		params, sc.rest.Protocol, sc.rest.Host, sc.rest.Port, sc.rest.LoginTimeout, sc.cfg.Authenticator.String())
-
-	respd, err := sc.rest.FuncPostAuth(ctx, sc.rest, params, headers, jsonBody, sc.rest.LoginTimeout)
+	respd, err := tryAuth(ctx, sc, samlResponse, proofKey, sessionParameters, clientEnvironment, params, headers)
```

Comment:
Could we rename the variable to be more descriptive, e.g. `authResp`?

Reply:
This name is widely used in this driver. I would stick to it.